app.js

```diff
@@ -0,0 +1,41 @@
+const books = [];
+
+const form = document.getElementById('book-form');
+function handleForm(event) {
+  event.preventDefault();
+}
+form.addEventListener('submit', handleForm);
+
+function renderBookList() {
+  const booksListContainer = document.getElementById('book-list-container');
+
+  const p = document.createElement('P');
+  const pText = document.createTextNode(books[books.length - 1]);
+  const removeButton = document.createElement('Button');
+  const buttonText = document.createTextNode('Remove');
+  const line = document.createElement('Hr');
+  const totalBooks = books.length;
+
+  p.appendChild(pText);
+  booksListContainer.appendChild(p);
+  removeButton.appendChild(buttonText);
+  removeButton.setAttribute('onclick', `removeBook(${[totalBooks - 1]})`);
+  booksListContainer.appendChild(removeButton);
+  booksListContainer.appendChild(line);
+}
+
+function addNewBook() {
+  const title = document.getElementById('title').value;
+  const author = document.getElementById('author').value;
```

Comment:
- [ ] This function  for adding new book should also follow the ES6 rule too.
- [ ] Optional you can concat the Title and Author here to make you adding book look good